docs/_source/docs/architecture.rst

```diff
@@ -11,7 +11,7 @@ our use of Sceptre at Cloudreach.
 Project layout
 --------------
 
-Sceptre’s nested StackGroups means that it is possible to store an entire
+Sceptre’s nested stack groups means that it is possible to store an entire
```

Comment:
Why was this changed here to be 2 separate words but I've seen some new instances below write them as `StackGroups` one word?

Reply:
Yeah, great question. Our docs and docstrings seem to be wholly inconsistent on this point. I think we could benefit from a "vocab guide" or something for contributing. It _does_ appear that "StackGroup" is more common than "stack group", though. I'll fix all occurrances in the files I've touched . 

![image](https://user-images.githubusercontent.com/77296393/144079916-f575d19e-b9a0-4099-bb38-123695609612.png)
![image](https://user-images.githubusercontent.com/77296393/144080003-69339f92-a213-43b2-b7da-71c4bd4d4fab.png)